src/librustc_codegen_ironox/metadata.rs

```diff
@@ -31,8 +32,30 @@ impl MetadataLoader for IronOxMetadataLoader {
         Ok(rustc_erase_owner!(buf))
     }
 
-    fn get_dylib_metadata(&self, _target: &Target, filename: &Path)
+    fn get_dylib_metadata(&self, target: &Target, filename: &Path)
         -> Result<MetadataRef, String> {
-        unimplemented!("get_dylib_metadata {:?}", filename);
+        let mut input_file =
+            File::open(filename)
```

Comment:
Minor nit. I'd have moved this up onto the previous line, but I don't feel strongly.

Reply:
You're right. Fixed!